spec/controllers/conference_controller_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
 describe ConferenceController do
-  let(:conference) { create(:conference,  splashpage: create(:splashpage, public: true)) }
+  let(:conference) { create(:full_conference,  splashpage: create(:splashpage, public: true)) }
```

Comment:
I know this seems convenient but do you really need all of this to test the schedule? I doubt it..